README.md

```diff
@@ -56,7 +56,7 @@ Radar.initialize('prj_test_pk_...', { /* options */ });
 
 Add the following script in your `html` file
 ```html
-<script src="https://js.radar.com/v4.1.12/radar.min.js"></script>
+<script src="https://js.radar.com/v4.2.0/radar.min.js"></script>
```

Comment:
Lets keep this as a patch bump: 4.1.13

Reply:
Just bumped to `4.1.14`. Not sure why the changes diff says the version on master is `4.1.12` when it's `4.1.13`